inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -50,53 +92,162 @@ class DeconvolutionLayerCPUTest : public testing::WithParamInterface<deconvLayer
 
         return result.str();
     }
+
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (int i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            ov::runtime::Tensor tensor;
+
+            if (i == 1) {
+                tensor = ov::runtime::Tensor(funcInput.get_element_type(), targetInputStaticShapes[i], outShapeData[inferRequestNum].data());
+            } else {
+                tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i], 2560, 0, 256);
+            }
+
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
+        inferRequestNum++;
+    }
+
+    void init_ref_function(std::shared_ptr<ov::Function> &funcRef, const std::vector<ov::Shape>& targetInputStaticShapes) override {
+        if (function->get_parameters().size() == 1) {
+            ngraph::helpers::resize_function(funcRef, targetInputStaticShapes);
+        } else {
+            funcRef = createGraph({targetInputStaticShapes[0]}, ngraph::helpers::InputLayerType::CONSTANT);
```

Comment:
Part of reference implementation can calculate output shapes (Interpolate, TopK, NonMaxSuppression etc.), but part can't like Deconvolution. So we need to rebuild the reference function with constant input, otherwise we can't execute reference Deconvolution

Reply:
So it is rather a WA. M.b. we need to add a comment to this code frame?

Reply:
added for deconv and group deconv